canonical/ubuntu.com
# Add global nav to new navigation

## Done

- added global nav 3.x

## QA

- Visit https://ubuntu-com-12191.demos.haus/
- See that "All Canonical" is now a menu item

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6105
